README.md

```diff
@@ -1,89 +1,88 @@
-## Türkçe Kaynaklar Listesi [![Build Status](https://img.shields.io/travis/fatihacet/turkcekaynaklar-com.svg?style=flat-square)](https://travis-ci.org/fatihacet/turkcekaynaklar-com)
+## Türkçe Kaynaklar Listesi
 
 Yazılım geliştirme ile ilgili özenle seçilmiş Türkçe kaynaklar listesi.
 
-### [https://turkcekaynaklar.com](https://turkcekaynaklar.com)
```

Comment:
| Expires On | Registered On | Updated On |
|--------------|-------------|--------------|
| 2025-02-27 | 2017-02-27 | 2024-04-09 |

@fatihacet ?

Reply:
I really couldn't understand this situation. VPN etc. to site content. I cannot access it with additional tools. I think there may be a problem with Deploy.

Reply:
I'll ask him next time we see each other. We can update or put it back accordingly afterwards. Marking as resolved for now.